api/schema/openai.go

```diff
@@ -117,6 +117,9 @@ type OpenAIRequest struct {
 	Functions    []grammar.Function `json:"functions" yaml:"functions"`
 	FunctionCall interface{}        `json:"function_call" yaml:"function_call"` // might be a string or an object
 
+	Tools     []grammar.Tool `json:"tools,omitempty" yaml:"tools"`
+	ToolsCall interface{}    `json:"tool_call,omitempty" yaml:"tool_call"`
```

Comment:
Sorry, my bad. It's [tool_choice](https://platform.openai.com/docs/api-reference/chat/create#chat-create-tool_choice).